src/imars3d/backend/workflow/engine.py

```diff
@@ -193,6 +222,8 @@ def run(self) -> None:
         self._dryrun()
         # initialize the registry of global parameters with the metadata
         self._registry = {k: v for k, v in self.config.items() if k not in ("name", "tasks")}
+        # This widget eats logs for some reason??
```

Comment:
`panel.widget` stuff does do that. Try rebase on `next` and you shouldn't need this line.